src/main/java/by/teachmeskills/data/DBUtil.java

```diff
@@ -0,0 +1,27 @@
+package by.teachmeskills.data;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBUtil {
+
+        private  static String dbUrl = "jdbc:mysql://localhost:3306/APP?serverTimezone=UTC";
```

Comment:
private static final String FIELD_NAME = "string"

missed "final" keyword and incorrect field name pattern